installing/installing_openstack/installing-openstack-installer-ovs-dpdk.adoc

```diff
@@ -0,0 +1,92 @@
+:_content-type: ASSEMBLY
+[id="installing-openstack-installer-ovs-dpdk"]
+= Installing a cluster on OpenStack that supports DPDK-connected compute machines
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-installer-ovs-dpdk
+:FeatureName: Installing a cluster on {rh-openstack} that supports DPDK-connected compute machines
+
+toc::[]
```

Comment:
lgtm now. Feel free to take a peek if you like.

Reply:
I'm still not seeing the TOC rendered in the [preview URL](https://deploy-preview-41484--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-ovs-dpdk.html) but that could be because I'm not very good at tracking multiple commits. 

![image](https://user-images.githubusercontent.com/47002677/157118655-dadb1610-8a0f-4258-a18d-b9359f25d2e6.png)

If you say it's good now, I believe it.

Reply:
@bobfuru Let's get maddeningly specific about this, as I'm curious and you're providing a screenshot. 

I'm not seeing anything on my preview that is different from the layout on prod. My brain is fried from trying to potty train my toddler over the last few days, so I'm not sure if I'm missing something really obvious. :) 

Could you mark up on the screenshot where you think the secondary TOC should be?

Reply:
I owe you a great big apology for my terrible lie, @maxwelldb. For no good reason, I was looking for the old school inline TOC that we had before Aidan gave us the righthand TOC. What you have is absolutely splendid. I'm sorry for wasting your time with this.  

My brain is still fried from potty training my toddlers and it's been years. Best of luck! And thanks for your patience with my feedback.

Reply:
@bobfuru No worries. I just wanted to make sure I hadn't gone UI blind or something. Thanks!